src/main/java/com/minecolonies/client/gui/AbstractWindowSkeleton.java

```diff
@@ -65,6 +65,7 @@ public void onButtonClicked(Button button)
      */
     public final void doNothing(Button ignored)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "ignored". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)